go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.4
+	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.1
```

Comment:
Can we do dependency/vendor changes in separate commit?

Reply:
I have pushed the latest changes before I do the splitting (just to be on safer side)... (=

Reply:
Completed the split that single commit into (a.) dependency updation and (b.) actual Snapshot changes
@umangachapagain  please take look...